src/pipecat/services/noisereduce.py

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
can we move the file to `processors/audio` please? this is not actually a third-party service but just a processor.